### ISSUE TITLE
to use conformer for acoustic scenes classification ?

### ISSUE THREAD

**Issue by Noumanijaz744** [sooftware/conformer#54]
thanks alot for the great work .can i use the mult-scale features aggregation conformer for acoustic scene classification ?

**Comment by sooftware** [comment]
Yes. You can add FC layers to the conformer's output and set it to the number of output scenes you want and train.
